examples/astro-upload-widget-preset/README.md

```diff
@@ -0,0 +1,98 @@
+# Astro & Cloduinary Upload Widget Example
+
+Example showing how to use the Cloudinary Upload Widget with an Astro project.
+
+Demo: 
+
+## 🧰 Using Upload Widget in an Astro project (Unsigned Uploads)
+
+> **Note**
+> This example uses unsigned uploads. This is not recommended for production use. See the [Cloudinary documentation](https://cloudinary.com/documentation/upload_widget#unsigned_uploads) for more information.
```

Comment:
i wouldn't add the "not for production" note there, while i certainly agree 99% of people should be using signed uploads, there are some use cases where the easy of unsigned uploads is an acceptable risk, especially when combined with options in the upload preset, such as moderation or other security features

Reply:
Shall I replace it with just "is not recommended"?